site/src/components/Callout/Callout.tsx

```diff
@@ -0,0 +1,39 @@
+import { type FC, type ReactNode } from "react";
```

Comment:
I don't see this implemented rn. Are we planning to use this in lieu of `<Alert />`?